libraries/AP_GPS/AP_GPS.h

```diff
@@ -168,6 +169,14 @@ class AP_GPS
         GPS_ROLE_MB_ROVER,
     };
 
+    // GPS Covariance Types matching ROS2 sensor_msgs/msg/NavSatFix
+    enum class GPS_Covariance_Type : uint8_t {
```

Comment:
```suggestion
    enum class CovarianceType : uint8_t {
```

GPS_ is redundant and the _ isn't what we usually use